<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassWord</class>
 <widget class="QDialog" name="PassWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../qshutdown.qrc">
    <normaloff>:/red_glasses</normaloff>:/red_glasses</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QPushButton" name="changePasswdButton">
     <property name="text">
      <string>&amp;Set password</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="passwdLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>passwdLineEdit</tabstop>
  <tabstop>changePasswdButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../qshutdown.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PassWord</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PassWord</sender>
   <signal>rejected()</signal>
   <receiver>PassWord</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
